build/ci/e2e/stack_versions.jenkinsfile

```diff
@@ -80,14 +80,14 @@ pipeline {
                         }
                     }
                 }
-                stage("7.5.0") {
+                stage("7.6.0") {
                     agent {
                         label 'linux'
                     }
                     steps {
                         checkout scm
                         script {
-                            runWith(lib, failedTests, "eck-75-${BUILD_NUMBER}-e2e", "7.5.0")
+                            runWith(lib, failedTests, "eck-76-${BUILD_NUMBER}-e2e", "7.6.0")
```

Comment:
This replaces our 7.5.0 E2E test by a 7.6.0 test.
We don't have a very structured way of choosing which versions should be automatically tested, in the meantime I think it's OK to not test them all?